src/Command/DebugEventsCommand.php

```diff
@@ -49,7 +50,6 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         if ($input->hasOption('groups') && $input->getOption('groups')) {
             $build = $this->getConfigBuild($config);
             $groups = array_keys($build);
-            ksort($groups);
```

Comment:
For what reason?

Reply:
Not need. array_keys returns already sorted array.

Reply:
https://3v4l.org/CmbjW

Reply:
Ksort is sorted by keys

Reply:
Maybe here need to use sort instead ksort?

Reply:
> Ksort is sorted by keys

What?

There should be a sorted list.

Reply:
See: https://3v4l.org/FAHBf

Reply:
Oh gotcha. You're right, there should be `sort` instead

Reply:
Added `sort`.